pkg/minishift/registration/redhat.go

```diff
@@ -70,8 +83,20 @@ func (registrator *RedHatRegistrator) Unregister(param *RegistrationParameters)
 		return err
 	} else {
 		if !strings.Contains(output, "not registered") {
-			if _, err := registrator.SSHCommand(
-				"sudo subscription-manager unregister"); err != nil {
+			unregisterCommand := "sudo subscription-manager unregister "
```

Comment:
:+1:, but you will need to make sure to set _RegistrationParameters_ in the stop command as well. This is another wrong assumption we had. @praveenkumar thought that for un-registration no proxy would be needed, but it need to make the actual un-registration request itself which needs to go through the proxy. We missed this the same way that we missed that registration was bypassing the proxy for registration altogether.